frontend/src/components/form/StepperNavigation.vue

```diff
@@ -39,20 +39,20 @@ function handlePrevClick() {
 <template>
   <div class="flex pt-4 justify-content-between">
     <Button
+      :style="{ visibility: props.prevDisabled ? 'hidden' : 'visible' }"
       class="px-4 py-1"
       outlined
       icon="pi pi-arrow-left"
       icon-class="text-xl"
-      :disabled="props.prevDisabled"
       @click="handlePrevClick"
     />
     <slot name="content" />
     <Button
+      :style="{ visibility: props.nextDisabled ? 'hidden' : 'visible' }"
```

Comment:
Same as above, preferred to not use style prop. Instead use create a class with the CSS attributes and dynamically insert the classes there.